### PR TITLE
Add bot PR/Issue comment support for custom commands

### PR DESCRIPTION
**Pull request by TheStonedTurtle**
Adds PR/Issue commenting support to the bot. Requires admin permissions and only supports other custom commands.

### New command format:
`!ghmsg <issue> <command>`
I.E.
`!ghmsg 7000 rejected`

This will take the output of `<command>` and add it as a comment to the PR/Issue, `<issue>`. Must specify the issue ID

### CODE REVIEW

**Comment by TheStonedTurtle** [comment]
Requested changes have been implemented

**Comment by deathbeam** [comment]
Pushed some changes to PR, can you re-test?

**Comment by TheStonedTurtle** [comment]
`!gh` and `!ghmsg` are still working as expected and the `!help` command output is displaying correctly.

Just to clarify the `!ghmsg` doesn't send a success message to the user/channel as the github bot should alert of the added comment.

**Comment by deathbeam** [comment]
Yeah, I assumed that is the case

**Comment by raiyni** [comment]
This should probably rate limit based on recently submitted commands to prevent double messages

File: lib/commands.js [hunk 1]
```diff
@@ -34,6 +39,50 @@ function noPermissions (message) {
...
+    return fetch(commentsURL, {
+      method: 'POST',
+      headers: {
+        'Authorization': `token ${config.githubToken}`,
+        'User-Agent': 'RuneLite-Bot',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        body: message
+      })
```
Comment (chivesrs) [inline lib/commands.js]: Given you have Content-Type set to JSON, do you need a JSON.stringify call? (I don't know the answer myself)

File: lib/commands.js [hunk 2]
```diff
@@ -34,6 +39,50 @@ function noPermissions (message) {
...
+    }).catch(e => log.debug(e))
+  })
+}
+
+function addCommandRan(command) {
+  commandsRan.push(command);
+  setTimeout(() => {
```
Comment (chivesrs) [inline lib/commands.js]: I don't know enough about JS async. Is it possible for two setTimeout callsbacks to trigger in such a manner that by the time you get to line 81, the value of `i` is incorrect? Or is there a guarantee that this function once entered will get to completion before another instance of the callback is ran?

**Pull request closed without merging**